app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/recordaudio/RecordAudioViewModel.kt

```diff
@@ -167,21 +175,11 @@ class RecordAudioViewModel @Inject constructor(
         audioMediaRecorder.release()
 
         if (state.originalOutputFile != null && state.effectsOutputFile != null) {
-            if (state.shouldApplyEffects) {
-                val result = AudioEffect(context)
-                    .applyEffectM4A(
-                        state.originalOutputFile!!.path,
-                        state.effectsOutputFile!!.path,
-                        AudioEffect.AVS_AUDIO_EFFECT_VOCODER_MED,
-                        true
-                    )
-
-                if (result > -1) {
-                    appLogger.i("[$tag] -> Audio file with effects generated successfully.")
-                } else {
-                    appLogger.w("[$tag] -> There was an issue with generating audio file with effects.")
-                }
-            }
```

Comment:
moved to usecase so this viewmodel could continue being tested throughout, as we can't mock AVS `AudioEffect`